session/session_test.go

```diff
@@ -4231,3 +4231,10 @@ func (s *testSessionSerialSuite) TestParseWithParams(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(sb.String(), Equals, "SELECT 3")
 }
+
+func (s *testSessionSerialSuite) TestNoIndexUsageCollector(c *C) {
+	do := s.dom
+	c.Assert(do.NeedCollectIndexUsage(), IsFalse)
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	c.Assert(tk.Se.ExistSessionIndexUsageCollector(), IsFalse)
+}
```

Comment:
This test seems not complete.
I suggest that we test it in this way:
1. enable this feature first, process some queries and then check `ExistSessionIndexUsageCollector` is not `nil`;
2. disable this feature, process the same queries and then check `ExistSessionIndexUsageCollector` is `nil`;